jax/lax/lax_control_flow.py

```diff
@@ -841,6 +838,13 @@ def _update_array(i, aval, xs, x):
   else:
     return lax.dynamic_update_index_in_dim(xs, x, i, 0)
 
+# TODO(mattjj): make scan a primitive
```

Comment:
I feel like I must be missing something here -- isn't that what `scan_p` in this file already is?

Reply:
Sloppy comment on my part! I guess that comment should read: make the scan impl execute as a single xla computation, rather than running Python.